builds/misc/templates/image-windows.yaml

```diff
@@ -3,7 +3,8 @@ parameters:
   imageName: ''
   namespace: 'microsoft'
   project: ''
+  version: ''
 
 steps:
-  - powershell: scripts/windows/build/Publish-DockerImage.ps1 -Name "${{ parameters.imageName }}" -Project "${{ parameters.project }}" -Version $(Build.BuildNumber) -Registry $(registry.address) -Namespace "${{ parameters.namespace }}" -Push
+  - powershell: scripts/windows/build/Publish-DockerImage.ps1 -Name "${{ parameters.imageName }}" -Project "${{ parameters.project }}" -Version ${{ parameters.version }} -Registry $(registry.address) -Namespace "${{ parameters.namespace }}" -Push
```

Comment:
Nit: Could you add `"` around the `${{ parameters.version }}` parameter please?

Reply:
Done. Thanks.